globus_automate_client/cli/flows.py

```diff
@@ -84,6 +79,12 @@ def _process_flow_input(flow_input: str, input_format) -> Mapping[str, Any]:
 
 app = typer.Typer(short_help="Manage Globus Automate Flows")
 
+_flows_env_var_option = typer.Option(
```

Comment:
Thanks for this, setting the same envvar option on every command was starting to grind my gears. At this point I'd probably be fine looking for it once in `os.environ` directly but we can save that for a future refactor.